app/views/layouts/_header.html.erb

```diff
@@ -20,7 +20,17 @@
                 <%= link_to "Home", root_path %>
               </li>
               <li>
-                <%= link_to "Projekty", projects_path %>
+                <%= link_to "Projekty1", projects_path %>
+              </li>
+              <li><a href="#">Projekty2</a>
+                <ul>
+                  <li>
+                    <%= link_to "Wizualizacje", wizualizacje_path %>
+                  </li>
+                  <li>
+                    <%= link_to "Realizacje", realizacje_path %>
+                  </li>
+                </ul>
               </li>
               <li>
                 <%= link_to "Kontakt", kontakt_path %>
```

Comment:
Lets create menu which consist of:
> `Home` `Wizualizacje` `Realizacje` `Kontakt` `Wyloguj`

as we dont have much items in menu and they can fit all together even for mobile devices, so we won't have to create separate menu for mobile devices.